<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLocalFields</class>
 <widget class="QDialog" name="DialogLocalFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,4,0,0">
   <item row="0" column="0">
    <widget class="QPushButton" name="newField">
     <property name="text">
      <string>new</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="addLocalField">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit1"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="text1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit2"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="text2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit3"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="text3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit4"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="text4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit5"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="text5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit6"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="text6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="lineEdit7"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="text7">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="lineEdit8"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="text8">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="lineEdit9"/>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="text9">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
